backend/.env.example

```
@@ -31,3 +31,8 @@ JWT_SECRET_KEY=secret
 # OPEN_AI
 #
 OPEN_AI_API_KEY=""
+#
+# GOOGLE AUTH
+#
+CLIENT_ID=""
+CLIENT_SECRET=""
```

To make it more clear
```suggestion
GOOGLE_CLIENT_ID=""
GOOGLE_CLIENT_SECRET=""
```